generic_hw_control_loop:
  loop_hz: 100
  cycle_time_error_threshold: 0.03

# Settings for ros_control hardware interface
hardware_interface:
  sim_control_mode: 0
  joints:
    - torso_1_joint
    - torso_2_joint
    - arm_right_1_joint
    - arm_right_2_joint
    - arm_right_3_joint
    - arm_right_4_joint
    - arm_right_5_joint
    - arm_right_6_joint
    - arm_right_7_joint